connectors/connectors-common-library/src/main/java/io/camunda/connector/common/utils/ResponseParser.java

```diff
@@ -1,53 +1,27 @@
 /*
  * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
- * under one or more contributor license agreements. See the NOTICE file
- * distributed with this work for additional information regarding copyright
- * ownership. Camunda licenses this file to you under the Apache License,
- * Version 2.0; you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
  */
```

Comment:
A question regarding a licensing. With a common library we apply Camunda OSS license, while HTTP REST connector was Apache 2.0. How is it resolved now? Is the HTTP REST still under Apache 2.0 despite few components being under under Camunda OSS?

@markfarkas-camunda @bastiankoerber @chillleader WDYT?

Reply:
Yeah I also have the same question. I think at least the common-library should definitely be Apache 2.0.